impeller/geometry/rect.h

```diff
@@ -226,6 +226,25 @@ struct TRect {
            p.y < bottom_;
   }
 
+  /// @brief  Returns true iff the provided point |p| is inside the
+  ///         closed-range interior of this rectangle.
+  ///
+  ///         Unlike the regular |Contains(TPoint)| method, this method
+  ///         considers all points along the boundary of the rectangle
+  ///         to be contained within the rectangle - useful for testing
+  ///         if vertices that define a filled shape would carry the
+  ///         interior of that shape outside the bounds of the rectangle.
+  ///         Since both geometries are defining half-open spaces, their
+  ///         defining geometry needs to consider their boundaries to
+  ///         be equivalent with respect to interior and exterior.
+  [[nodiscard]] constexpr bool ContainsInclusive(const TPoint<Type>& p) const {
```

Comment:
Can you a few test cases to https://github.com/flutter/engine/blob/main/impeller/geometry/geometry_unittests.cc for this? Just so its easier to find compared to knowing the specific DL tests its used in

Reply:
You mean https://github.com/flutter/engine/blob/2f406fe7382df9a34ba216f24a4cdee433b5fcae/impeller/geometry/rect_unittests.cc#L2107